clamav/clamav.lua

```
@@ -1,318 +1,297 @@
+    end
+    
+    current_pos = next_boundary_pos + #boundary
+    
+    -- Check for end boundary (ends with --)
+    if body:sub(current_pos, current_pos + 1) == "--" then
+      break
+    end
+  end
+  
+  -- Parse each section into headers and data
+  for i, section in ipairs(sections) do
```

Luacheck (1.2.0)

[warning] unused loop variable 'i'

(W213)